src/device-registry/utils/create-airqloud.js

```diff
@@ -91,33 +95,50 @@ const createAirqloud = {
   },
   create: async (request) => {
     try {
-      const { body } = request;
-      const { tenant } = request.query;
-      const { location_id } = request.body;
+      const { body, query } = request;
+      const { tenant } = query;
```

Comment:
enhanced AirQloud creation with logic for adding the AirQloud center in the process